components/browser/state/src/main/java/mozilla/components/browser/state/state/recover/RecoverableTab.kt

```diff
@@ -33,6 +33,7 @@ import mozilla.components.concept.storage.HistoryMetadataKey
  * @property private If tab was private.
  * @property historyMetadata The last [HistoryMetadataKey] of the tab.
  * @property source The last [Source] of the tab.
+ * @property index The index of the tab at the time of removal.
```

Comment:
nit: Let's not mention "removal" here since that is only one use case for recovering. Let's say something like "index the tab should be restored at"

Reply:
Thanks for catching these! I totally missed updating the removal index verbiage after changing the class variable name.